docs/en/admins/15_extensions.md

```diff
@@ -0,0 +1,41 @@
+# Extensions
+
+Not every feature is relevant for the most users, but it some special features are relevant for the user. That is why FreshRSS is extendable.
+
+There are a bunch of "official" extensions (supported and published by FreshRSS development team and community) and "Community" extensions (developed and published individually by thiry-party).
+
+## Extension repositories
+
+Most known extensions are listed in the front end: see configuration menu `Configuration/Extensions`.
+
+## How to install
+
+Upload the folder (f.e. `CustomCSS`) of your chosen extension into your `./extensions` directory.
+
+Result: Content of `./extensions/CustomCSS/` has f.e. `extension.php`, `metadata.json`, `configure.php`, `README.md` files and the folders `i18n` and `static`
+
+Important: Do not delete or overwrite the existing files `./extensions/.gitignore` and `./extensions/README.md` 
+
+## How to enable/disable and manage
+
+See in the front end: configuration menu `Configuration/Extensions`
+
+### User extensions
+
+Every user has to manage the extensions by them self. The configuration via the cog icon is valid only for the user, not for other users. 
```

Comment:
A cog is just one little tooth. The whole thing is a cogwheel or a gear. I prefer gear.
```suggestion
Every user has to manage the extensions by themselves. Configuration via the gear icon is valid only for that user, not for other users. 
```